drydock/values.yaml

```diff
@@ -0,0 +1,49 @@
+
+# Copyright 2017 The Openstack-Helm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file provides defaults for drydock
+
+replicas:
+  drydock: 3
+
+images:
+  drydock: docker.io/srwilkers/drydock:v0.1.0
```

Comment:
@alanmeadows of course.  I'm in the process of putting together a WIP pull request for the Dockerfiles repo that can be updated as additional functionality is added to Drydock.